.dockerignore

```diff
@@ -0,0 +1,32 @@
+# Object files
```

Comment:
Hierboven even een "# When using docker in a build context, a COPY need not copy build products." ofzo.

Ik snapte niet meteen waar de dockerignore voor was.